org-code-javabuilder/validation/src/test/java/org/code/validation/ClientMessageHelper.java

```diff
@@ -0,0 +1,10 @@
+package org.code.validation;
+
+import org.code.protocol.ClientMessage;
+import org.code.protocol.ClientMessageType;
+
+public class ClientMessageHelper extends ClientMessage {
```

Comment:
For my understanding, is this just for tests? Or will it be something more generic than when we extend `ClientMessage` elsewhere? I would have expected `ValidationMessage` or something?

Reply:
this is just for tests, so we don't need to reference classes like `SystemOutMessage` and `TheaterMessage`

Reply:
👍 maybe a comment to that effect then?

Reply:
done!